js/fetch.ts

```diff
@@ -97,59 +99,127 @@ export const Headers = DomIterableMixin<string, string, typeof HeadersBase>(
   headerMap
 );
 
-class FetchResponse implements domTypes.Response {
+class Body implements domTypes.Body, domTypes.ReadableStream, io.Reader {
+  bodyUsed = false;
+  readonly body: domTypes.ReadableStream | null = null; // TODO
+  private _buffering = false;
+  private _bodyWaiter: Resolvable<ArrayBuffer>;
+  private _chunks: Uint8Array[] = [];
+  private _data: Uint8Array | null = null;
+  readonly locked: boolean = false; // TODO
+
+  constructor(readonly rid: number, readonly contentType: string) {
+    this._bodyWaiter = createResolvable();
+  }
+
+  private async _bufferBody(): Promise<void> {
+    if (!this._buffering) {
+      this._buffering = true;
+      try {
+        while (true) {
+          const b = new Uint8Array(64 * 1024);
+          const r = await read(this.rid, b);
+          if (r.nread > 0) {
+            this._chunks.push(b.subarray(0, r.nread));
+          }
+          if (r.eof) {
+            break;
+          }
+        }
+        this._data = concatUint8Array(this._chunks);
+        this._chunks = [];
+        this._bodyWaiter.resolve(this._data!.buffer as ArrayBuffer);
+      } catch (e) {
+        this._bodyWaiter.reject(e);
+      } finally {
+        this._buffering = false;
+      }
+    }
+  }
+
+  async arrayBuffer(): Promise<ArrayBuffer> {
+    this._bufferBody();
+    return this._bodyWaiter;
+  }
+
+  async blob(): Promise<domTypes.Blob> {
+    const arrayBuffer = await this.arrayBuffer();
+    return new DenoBlob([arrayBuffer], {
+      type: this.contentType
+    });
+  }
+
+  async formData(): Promise<domTypes.FormData> {
+    notImplemented();
+    return {} as domTypes.FormData;
+  }
+
+  // tslint:disable-next-line:no-any
+  async json(): Promise<any> {
```

Comment:
Use unknown.

Reply:
That results in this error:
```
/Users/rld/src/deno/js/fetch_test.ts:8:15 - error TS2571: Object is of type 'unknown'.

8   assertEqual(json.name, "deno");
                ~~~~
```

Reply:
`unknown` is assignable to any type... it just keeps people from using it without putting some sort of type cast on it.  You could simple do:

```
const json: any = await fetch(...).json();
// or
const json: { [v: string]: string; } = await fetch(...).json();
```

But considering we haven't forced strict mode on Deno user land, this seems like an overly strict construct to for on users now.